youtube_dl/extractor/keezmovies.py

```diff
@@ -55,7 +55,7 @@ def _extract_info(self, url):
         encrypted = False
 
         def extract_format(format_url, height=None):
-            if not isinstance(format_url, compat_str) or not format_url.startswith('http'):
+            if not isinstance(format_url, compat_str):
```

Comment:
No.

Reply:
@dstftw - Can you please elaborate?  I believe the provided `format_url` value no longer contains the http/https prefix in the page markup.  Thank you!